pocs/state/states/default/pointing.py

```diff
@@ -34,44 +31,24 @@ def on_enter(event_data):
         fits_headers['POINTING'] = 'True'
         pocs.logger.debug("Pointing headers: {}".format(fits_headers))
 
+        primary_camera = pocs.observatory.primary_camera
+
         # Loop over maximum number of pointing iterations
         for img_num in range(num_pointing_images):
-            camera_events = dict()
-
-            # Take pointing image with primary camera
-            for cam_name, camera in pocs.observatory.cameras.items():
-                if camera.is_primary:
-                    pocs.logger.debug("Exposing for camera: {}".format(cam_name))
-                    try:
-                        # Start the exposures
-                        camera_event = camera.take_observation(
-                            observation,
-                            fits_headers,
-                            exp_time=exptime,
-                            filename='pointing{:02d}'.format(img_num)
-                        )
-
-                        camera_events[cam_name] = camera_event
-
-                    except Exception as e:
-                        pocs.logger.error("Problem waiting for images: {}".format(e))
-
-            # Wait for images to complete
-            wait_time = 0.
-            while not all([event.is_set() for event in camera_events.values()]):
-                pocs.check_messages()
-                if pocs.interrupted:
-                    pocs.say("Observation interrupted!")
-                    break
-
-                pocs.logger.debug('Waiting for images: {} seconds'.format(wait_time))
-                pocs.status()
+            pocs.logger.debug("Taking pointing image {}/{} on: {}",
+                              img_num, num_pointing_images, primary_camera)
 
-                if wait_time > timeout:
-                    raise error.Timeout("Timeout waiting for pointing image")
+            # Start the exposure
+            camera_event = primary_camera.take_observation(
```

Comment:
Is the type of camera_event `list of events`? If so, please rename as camera_events, observation_done_events, or some such.

Reply:
Here it's just a single event. The `wait_for_events` handles either a single Event or a list.